compiler/rustc_metadata/src/locator.rs

```diff
@@ -340,17 +338,9 @@ impl<'a> CrateLocator<'a> {
             },
             hash,
             extra_filename,
-            target: if is_host { &sess.host } else { &sess.target },
-            triple: if is_host {
-                TargetTriple::from_triple(config::host_triple())
-            } else {
-                sess.opts.target_triple.clone()
-            },
-            filesearch: if is_host {
-                sess.host_filesearch(path_kind)
-            } else {
-                sess.target_filesearch(path_kind)
-            },
+            target: &sess.target,
```

Comment:
I'm so confused. The callsite is always `false` and there was a casual `is_host` branch here? Archaeology didn't reveal much for why this came to be, oh well.

Reply:
But apparently this seems to work since 5 years ago when `is_host` was introduced, then I guess it must've worked because nobody complained about it 🤷

Reply:
In https://github.com/rust-lang/rust/pull/66496/files

```rs
/// Look for a plugin registrar. Returns its library path and crate disambiguator.
pub fn find_plugin_registrar(
```

so I think this is correct since we ripped out compiler plugin support some time ago.